package.json

```diff
@@ -8,13 +8,13 @@
     "node": ">=4.2.0"
   },
   "dependencies": {
-    "babel-core": "^6.14.0",
-    "babel-loader": "^6.2.5",
-    "babel-preset-es2015": "6.14.0",
-    "express": "^4.14.0",
-    "rimraf": "^2.5.4",
-    "start-server-webpack-plugin": "^2.0.1",
-    "webpack": "^2.1.0-beta"
+    "babel-core": "^6.25.0",
+    "babel-loader": "^7.0.0",
+    "babel-preset-latest": "^6.24.1",
```

Comment:
actually, `latest` is deprecated, instead, use `babel-preset-env` instead; in this case, using the setting `targets: {node: "current"}}`